biothings/web/query/engine.py

```diff
@@ -139,6 +142,13 @@ async def execute(self, query, **options):
                     raise RawResultInterrupt(res)
 
                 if not res["hits"]["hits"]:
+                    scroll_id = str(query.data) if query.data else "None"
+                    try:
+                        await self.client.clear_scroll(scroll_id=query.data)
```

Comment:
if you already have a `scroll_id` variable before this line, should we just pass `scroll_id` here?